src/graph/executor/algo/IsomorExecutor.cpp

```diff
@@ -12,77 +12,61 @@ namespace graph {
 folly::Future<Status> IsomorExecutor::execute() {
    // TODO: Replace the following codes with subgraph matching. Return type.
    SCOPED_TIMER(&execTime_);
-   auto* subgraph = asNode<Subgraph>(node());
+   auto* isomor = asNode<Isomor>(node());
    DataSet ds;
-   ds.colNames = subgraph->colNames();
+   ds.colNames = isomor->colNames();
 
-   uint32_t steps = subgraph->steps();
-   const auto& currentStepVal = ectx_->getValue(subgraph->currentStepVar());
-   DCHECK(currentStepVal.isInt());
-   auto currentStep = currentStepVal.getInt();
-   auto resultVar = subgraph->resultVar();
+   auto iterDV = ectx_->getResult(isomor->getdScanVOut()).iter();
+   auto iterQV = ectx_->getResult(isomor->getqScanVOut()).iter();
+   auto iterDE = ectx_->getResult(isomor->getdScanEOut()).iter();
+   auto iterQE = ectx_->getResult(isomor->getqScanEOut()).iter();
 
-   auto iter = ectx_->getResult(subgraph->inputVar()).iter();
-   auto gnSize = iter->size();
+   unsigned int v_count = iterDV->size();
+   unsigned int l_count = iterDV->size();
+   unsigned int e_count = iterDE->size();
 
-   ResultBuilder builder;
-   builder.value(iter->valuePtr());
+   unsigned int *offset = new unsigned int[v_count + 2];
+   unsigned int *neighbors = new unsigned int[e_count * 2];
+   unsigned int *labels = new unsigned int[l_count];
+   // load data vertices id and tags
+   while (iterDV->valid()) {
+     const auto v = iterDV->getColumn(nebula::kVid);  // check if v is a vertex
```

Comment:
what are v and v1. Please use meaningful names.

Reply:
I change it to the new version in the code, v is the content of the vertex id while v2 is the content of the label id.